pandas/computation/tests/test_eval.py

```diff
@@ -609,6 +609,15 @@ def test_scalar_unary(self):
         self.assertEqual(
             pd.eval('+False', parser=self.parser, engine=self.engine), +False)
 
+    def test_unary_in_array(self):
+        assert_numpy_array_equal(
+            pd.eval('[-True, True, ~True, +True,'
+                    '-False, False, ~False, +False,'
+                    '-37, 37, ~37, +37]'),
+            np.array([-True, True, ~True, +True,
+             -False, False, ~False, +False,
+             -37, 37, ~37, +37]))
```

Comment:
can you add in the op original example as well (could locate where some of other eval tests are, in test_frame IIRC)

add in the issue number here as well